src/gui/hist_dialog.c

```diff
@@ -244,7 +244,8 @@ int dt_gui_hist_dialog_new(dt_history_copy_item_t *d, int imgid, gboolean iscopy
   gtk_tree_selection_set_mode(gtk_tree_view_get_selection(GTK_TREE_VIEW(d->items)), GTK_SELECTION_SINGLE);
   gtk_tree_view_set_model(GTK_TREE_VIEW(d->items), GTK_TREE_MODEL(liststore));
 
-  GdkPixbuf *is_active_pb = dt_draw_paint_to_pixbuf(GTK_WIDGET(dialog), 10, dtgtk_cairo_paint_switch);
+  GdkPixbuf *is_active_pb = dt_draw_paint_to_pixbuf(GTK_WIDGET(dialog), 10, CPF_FOCUS, dtgtk_cairo_paint_switch_active);
+  GdkPixbuf *is_inactive_pb = dt_draw_paint_to_pixbuf(GTK_WIDGET(dialog), 10, 0, dtgtk_cairo_paint_switch);
```

Comment:
For me this one should be for active.
The inactive one should be the same dimmed.

Reply:
Any idea on how to dim a cell in a treeview?

Reply:
Ok, found a way:

![image](https://user-images.githubusercontent.com/467069/149406812-3dfc9a0c-e4f3-44a2-b051-e7d633f3d694.png)

Better for you?

Reply:
Ok, found a way:

![image](https://user-images.githubusercontent.com/467069/149406893-906fad80-6547-47d9-80ba-c42a0ca3866f.png)

Better for you?

Reply:
Trying to find the way ...
Perfect for me !